appveyor.yml

```diff
@@ -51,7 +51,9 @@ install:
   - cmd: R -e "packageVersion('knitr'); packageVersion('rmarkdown'); packageVersion('testthat'); packageVersion('e1071'); packageVersion('survival')"
 
 build_script:
-  - cmd: mvn -DskipTests -Psparkr -Phive package
+  # '-Djna.nosys=true' is required to avoid kernel32.dll load failure.
+  # See SPARK-28759.
+  - cmd: mvn -DskipTests -Psparkr -Phive -Djna.nosys=true package
```

Comment:
do we need to document this as "building spark on windows" then? ;)

Reply:
Oh, do you mean https://github.com/apache/spark/blob/master/R/WINDOWS.md#building-sparkr-on-windows ? Yes, I guess we better note it.

Actually, I have some more additional and required information to document there. Building Spark on Windows requires `bash` (by Windows 11 or Cygwin) due to here:

https://github.com/apache/spark/blob/eed6de1a6533f259cbd60654d3e537ff7088f296/project/SparkBuild.scala#L516

Reply:
Let me document this it later in few days.